libs/community/langchain_community/vectorstores/opensearch_vector_search.py

```diff
@@ -618,7 +615,6 @@ def add_embeddings(
         text_embeddings: Iterable[Tuple[str, List[float]]],
         metadatas: Optional[List[dict]] = None,
         ids: Optional[List[str]] = None,
-        bulk_size: int = 500,
```

Comment:
breaking change - can we keep this, and use the passed-in as an override? can still default to the `self.bulk_size` behavior if it's None